ControlFlow/Program.cs

```diff
@@ -15,22 +15,47 @@ public static class ControlFlow
         // be validated).  Negative numbers should return "Invalid".
         public static string AreYouValid(int number)
         {
-            return default;
+            if (number > 1 && number < 10)
+            {
+                return "Valid";
+            }
+            else
+            {
+                return "Invalid";
+            }
         }
 
         // 1.2 - Write a method which takes two integers and returns the maximum of the two. If they
         // are the same, return either one.
         public static int Maximum(int number1, int number2)
         {
-            return default;
+            if (number1 >= number2)
+            {
+                return number1;
+            }
+            else
+            {
+                return number2;
+            }
         }
 
         // 1.3 - Write a method which takes two integers, the width and height of an image. Then
         // return a string to tell if the image is "Landscape" or "Portrait". Return "Square" if
         // they are equal. Assume non-negative values.
         public static string LandscapeOrPortrait(int width, int height)
         {
-            return default;
+            if (width > height)
+            {
+                return "Landscape";
+            }
+            if (height > width)
+            {
+                return "Portrait";
+            }
+            else
+            {
+                return "Square";
+            }
         }
```

Comment:
I recommend using fewer return statements by using a variable to change the specific return value.

```
var result = "Landscape";
result = "Portrait";
result = "Square";
return result;
```